libimagstore/src/store.rs

```
@@ -125,7 +125,38 @@ impl Store {
 
     /// Creates the Entry at the given location (inside the entry)
     pub fn create(&self, entry: Entry) -> Result<()> {
-        unimplemented!();
+        use std::fs::create_dir_all;
+        use std::io::Write;
+        use std::ops::Deref;
+
+        let path = entry.location().clone();
+        if !path.is_absolute() {
+            return Err(StoreError::new(StoreErrorKind::InvalidStorePath, None));
+        }
+
+        let dirpath = {
+            if path.file_name().is_some() {
+                let mut newpath = self.location.clone();
+                newpath.push(path.parent().unwrap().to_path_buf()); // TODO: unwrap() safe here?
+                newpath
+            } else {
+                unimplemented!()
```

Do you plan on implementing something? Why not return an error?